src/pages/library/DataBrowserDetails.js

```diff
@@ -43,7 +43,7 @@ const MetadataDetailsComponent = ({ dataObj, name }) => {
     div({ style: { display: 'flex', width: '100%', flexWrap: 'wrap' } }, [
       div({ style: styles.attributesColumn }, [
         h3({ style: styles.headers }, ['Data release policy']),
-        h(DatasetReleasePolicyDisplayInformation, { dataUsePermission: dataObj['TerraDCAT_ap:hasDataUsePermission'] })
+        h(DatasetReleasePolicyDisplayInformation, dataObj)
```

Comment:
If the component `DatasetReleasePolicyDisplayInformation` only uses the `TerraDCAT_ap:hasDataUsePermission` property, is there a way to have it accept only the value of the property instead of requiring the name too? Maybe one way would be to make `DatasetReleasePolicyDisplayInformation` into a lambda that accepts the permission string and then returns the component.

I'm looking at how this is called elsewhere and it seems awkward to have to explicitly provide `TerraDCAT_ap:hasDataUsePermission` when this is the only property that's used.

Reply:
Lets chat about this at standup. I have some thoughts about this that would be better talked through with words, I think.